core/Pos/Core/Types.hs

```diff
@@ -102,6 +103,15 @@ import           Pos.Data.Attributes        (Attributes)
 -- Address
 ----------------------------------------------------------------------------
 
+-- | Hash used to identify address.
+type AddressHash = AbstractHash Blake2b_224
+
+-- | Stakeholder identifier (stakeholders are identified by their public keys)
+type StakeholderId = AddressHash PublicKey
+
+-- | Alias for set of stakeholders.
+type Stakeholders = HashSet StakeholderId
```

Comment:
Btw, I don't like this alias. It's quite subjective. I can't say that we never want to use `[StakeholderId]` or `Vector StakeholderId` which can be called `Stakeholders` too.
For me such aliases only make sense if we know for sure that the only reasonable container is set.
Anyway, it's minor.